components/core/CMakeLists.txt

```diff
@@ -73,6 +73,26 @@ elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     endif ()
 endif ()
 
+# Java 11 is a dependency for antlr. The way we install java
+# with brew doesn't also make it the default version of java on the system. 
+# Set JAVA_HOME to java 11 install location here.
+if (APPLE)
+    set (ENV{JAVA_HOME} "/usr/local/opt/openjdk@11/")
+endif ()
+
+# Find and setup Antlr Library
+# We build and link to the static library
+set(ANTLR4_TAG 4.11.1)
+add_definitions(-DANTLR4CPP_STATIC)
```

Comment:
Can you check if there's a way to make this less global? E.g., maybe we can apply it to just clg using target_compile_definitions, or maybe it's only necessary for the ANTLR runtime and we can apply it with a better FindANTLR in the future.